.github/workflows/ci-test.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        k8sVersion: ["1.25.x", "1.26.x", "1.27.x", "1.28.x"]
+        k8sVersion: ["1.26.x", "1.27.x", "1.28.x", "1.29.x"]
```

Comment:
Updating to supported versions matrix of AKS. 

We don't have to merge this right away as I saw some probelms with 1.25 we could fix if we care about unsupported AKS Versions

Reply:
Good question here. I could go either way, but unless we decide to make a practice of removing tests for AKS unsupported version than we should probably fix until then.

Reply:
Added back `1.25.x` as fix was found and pushed.